spec/monthday.html

```diff
@@ -294,18 +294,6 @@ <h1>ValidateMonthDay ( _month_, _day_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-temporal-balancemonthday" aoid="BalanceMonthDay">
-      <h1>BalanceMonthDay ( _month_, _day_ )</h1>
```

Comment:
Looks like we should add `RegulateTemporalMonthDay` that will call it.

Reply:
...which is a problem you identified as issue #444 :smile: 

I guess the best course of action here is to just merge this pull request but drop the commit that removes BalanceMonthDay, and fix that in #444.